decidim-core/app/views/decidim/processes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<main class="wrapper">
+  <% if hero_processes.any? %>
+    <section id ="highlighted-processes" class="row section">
+      <h1 class="section-heading"><%= t(".hero_processes", scope: "layouts") %></h1>
+        <% hero_processes.each do |hero_process| %>
+          <%= render partial: "hero_process", locals: { hero_process: hero_process } %>
+        <% end %>
+    </section>
+  <% end %>
+
+  <section id="processes-grid" class="section row collapse">
+    <%= render partial: "order_by_processes" %>
+    <div class="row small-up-1 medium-up-2 large-up-3 card-grid">
+      <% participatory_processes.each do |participatory_process| %>
+        <%= render partial: "process", locals: { participatory_process: participatory_process } %>
```

Comment:
Also, I think it would be beneficial to add counter caches to the organization (both hero and regular processes), this way we avoid queries by checking if there are any of them (and to display the count) since we always have the organization loaded /cc @josepjaume @beagleknight